src/service/viewer-impl.js

```diff
@@ -89,8 +89,8 @@ export class ViewerImpl {
     /** @private {number} */
     this.prerenderSize_ = 1;
 
-    /** @private {!Object<string, !Observable<!JsonObject>>} */
-    this.messageObservables_ = map();
```

Comment:
Chaning `map` to `dict` isn't equivalent. `map` creates prototype-less objects, `dict` is just a typecast.

Reply:
should be compatible now that `dict` also returns a prototype-less object